docker/docker-json-config/agent.d/docker.json

```diff
@@ -3,10 +3,9 @@
 // docker container log directory (which must be mounted into the
 // agent's container).
 {
- monitors: [
-   {
-    module: "scalyr_agent.builtin_monitors.docker_monitor",
-    docker_raw_logs: true
```

Comment:
Safe to remove? Does it impact existing customers?

Reply:
It won't impact existing customers.  Only customers who use the new version will get this config file, and for them the default will be true.